app/controllers/users_controller.rb

```diff
@@ -1,12 +1,15 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
phải check login cho action show nữa chứ em, mà chuyển sang dùng except cho gọn code nè
```suggestion
  before_action :logged_in_user, except: %i(new create)
```